examples/MyModule/CMakeLists.txt

```diff
@@ -187,10 +184,14 @@ execute_process(
 # on OS X
 set( CMAKE_MACOSX_RPATH ON )
 
-# Install all stuff to NEST's install directories.
-set( CMAKE_INSTALL_LIBDIR ${NEST_LIBDIR}/nest CACHE STRING "object code libraries (lib/nest or lib64/nest or lib/<multiarch-tuple>/nest on Debian)" FORCE )
-set( CMAKE_INSTALL_DOCDIR ${NEST_INSTALL_DOCDIR} CACHE STRING "documentation root (DATAROOTDIR/doc/nest)" FORCE )
-set( CMAKE_INSTALL_DATADIR ${NEST_INSTALL_DATADIR} CACHE STRING "read-only architecture-independent data (DATAROOTDIR/nest)" FORCE )
+if ( "${CMAKE_INSTALL_PREFIX}" STREQUAL "/usr/local" )
```

Comment:
Comments and spacing look weird to me, could you change to

```cmake
# Use the NEST_INSTALL_* variables as CMAKE_INSTALL_*, if not set explicitly.
if ( "${CMAKE_INSTALL_PREFIX}" STREQUAL "/usr/local" )
  set( CMAKE_INSTALL_PREFIX "${NEST_INSTALL_PREFIX}" CACHE STRING "Install path prefix, prepended onto install directories." FORCE )
  set( CMAKE_INSTALL_LIBDIR ${NEST_LIBDIR}/nest CACHE STRING "object code libraries (lib/nest or lib64/nest or lib/<multiarch-tuple>/nest on Debian)" FORCE )
  set( CMAKE_INSTALL_DOCDIR ${NEST_INSTALL_DOCDIR} CACHE STRING "documentation root (DATAROOTDIR/doc/nest)" FORCE )
  set( CMAKE_INSTALL_DATADIR ${NEST_INSTALL_DATADIR} CACHE STRING "read-only architecture-independent data (DATAROOTDIR/nest)" FORCE )
endif ()
```

?

I am also a little skeptical about what happens if ``"${CMAKE_INSTALL_PREFIX}"`` is not ``"/usr/local"``... how do we know all other ``CMAKE_INSTALL_*`` variables are correct? Shouldn't we check all?

Reply:
> ... "${CMAKE_INSTALL_PREFIX}" is not "/usr/local" ...

When `CMAKE_INSTALL_PREFIX` is not set with `cmake -DCMAKE_INSTALL_PREFIX=...` then it is set to `/usr/local`. This is the same reasoning we apply to check if nest will be installed to the default location: https://github.com/nest/nest-simulator/blob/master/CMakeLists.txt#L97 There was also an issue discussing this behaviour in NEST, but i cannot find it anymore ...

The other `CMAKE_INSTALL_*` are set with `include( GNUInstallDirs )` later, see [here](https://cmake.org/cmake/help/v3.4/module/GNUInstallDirs.html)

I will apply the requested changes.